designs/io_placer/config.tcl

```diff
@@ -0,0 +1,128 @@
+# SPDX-FileCopyrightText: 2020 Efabless Corporation
```

Comment:
Also... why config.tcl? We'd moved on to .json for almost all designs. See `manual_macro_placement_test` for an example.

Reply:
I used the config.tcl as it was an exact replica from the caravel_upw design which still used tcl, however I changed the config file to JSON in this commit.